airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryStagingOperations.java

```diff
@@ -7,42 +7,77 @@
 import com.google.cloud.bigquery.Schema;
 import com.google.cloud.bigquery.TableId;
 import io.airbyte.integrations.destination.record_buffer.SerializableBuffer;
-import io.airbyte.protocol.models.v0.DestinationSyncMode;
 import java.util.List;
 
 /**
  * This interface is similar to
  * {@link io.airbyte.integrations.destination.s3.BlobStorageOperations}.
+ *
+ * <p>
+ * Similar interface to {@link io.airbyte.integrations.destination.jdbc.SqlOperations}
```

Comment:
could maybe be combined into the comment above if it's similar to both interfaces?

Reply:
meta question: if the interfaces are similar should they be the same?